input/system/tembo/system.go

```diff
@@ -0,0 +1,165 @@
+package tembo
+
+import (
+	"encoding/json"
+	"github.com/pganalyze/collector/config"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+type Response struct {
+	Status string `json:"status"`
+	Data   Data   `json:"data"`
+}
+
+type Data struct {
+	ResultType string   `json:"resultType"`
+	Result     []Result `json:"result"`
+}
+
+type Result struct {
+	Metric Metric        `json:"metric"`
+	Value  []interface{} `json:"value"`
+}
+
+type Metric struct {
+	Name                  string `json:"__name__"`
+	Endpoint              string `json:"endpoint"`
+	Instance              string `json:"instance"`
+	Job                   string `json:"job"`
+	MetricsPath           string `json:"metrics_path"`
+	Namespace             string `json:"namespace"`
+	Node                  string `json:"node"`
+	PersistentVolumeClaim string `json:"persistentvolumeclaim"`
+	Service               string `json:"service"`
+}
+
+// GetSystemState - Gets system information for a Tembo Cloud instance
+func GetSystemState(config config.ServerConfig, logger *util.Logger) (system state.SystemState) {
+	headers := map[string]string{
+		"Authorization": "Bearer " + config.TemboAPIToken,
+		"accept":        "application/json",
+	}
+
+	client := http.Client{}
+
+	// Get CPU usage percentage
+	query := "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{ namespace=\"" + config.TemboNamespace + "\"}) / sum(kube_pod_container_resource_requests{job=\"kube-state-metrics\",  namespace=\"" + config.TemboNamespace + "\", resource=\"cpu\"})"
+	cpuUsage, err := getFloat64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
```

Comment:
As discussed both the `data-1.use1` and the `tembo.io` parts need to be configurable (for different regions, and dev/staging envs).

Reply:
I added a new env var for `TEMBO_METRICS_API_URL`:
- https://github.com/pganalyze/collector/pull/509/commits/397c4a4641fdfd677b3b92f0e2b664ed0bf44c8c
- https://github.com/pganalyze/collector/pull/509/commits/17dc38f92d47766a4fd768fce3944df3be9e4d8e

Let me know what you think of this approach